sdk/script/management-sdk.ts

```diff
@@ -257,8 +251,15 @@ class AccountManager {
         });
     }
 
-    public promote(appName: string, sourceDeploymentName: string, destDeploymentName: string): Promise<void> {
-        return this.post(urlEncode `/apps/${appName}/deployments/${sourceDeploymentName}/promote/${destDeploymentName}`, /*requestBody=*/ null, /*expectResponseBody=*/ false)
+    public patchRelease(appName: string, deploymentName: string, packageInfo: PackageInfo): Promise<void> {
```

Comment:
I am not entirely sure, if that is the right approach. I agree, having it as an argument makes SDK more readable and easy to understand. However, its possible that the label value could be null, and we aren't addressing it in the method directly, it will be a placeholder which we append to the `updateMetadata` object. But then again the example you mentioned does look more compelling, barring that it could be null. I will make the change.